GTBitsOfGood/angels-among-us
# Onboarding Bugs

## Onboarding Bugs

Issue Number(s): #68 

What does this PR change and why?
- Tooltips changed to popovers
- Popovers have restricted height and are scrollable
- Change wording on onboarding bugs
- Persist user preferences to database after hitting get started button on last slide of onboarding sequence

### Checklist


### Critical Changes

- None

### Related PRs

- None

### Testing

1. Check popovers are scrollable when they overflow
2. Check that my user document is updated with the correct preferences after I go through onboarding process
